set.mm

```diff
@@ -147746,6 +147746,473 @@ of an open range of nonnegative integers (of length equal to the length of
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Prefixes of a word
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+
+  In ~ https://www.allacronyms.com/prefix/abbreviated , "pfx" is proposed as
+  abbreviation for "prefix".  Regarding the meaning of "prefix", it is
+  different in computer science (automata theory/formal languages) compared
+  with linguistics: in linguistics, a prefix has a meaning (see Wikipedia
+  "Prefix" ~ https://en.wikipedia.org/wiki/Prefix ), whereas in computer
+  science, a prefix is an arbitrary substring/subword starting at the
+  beginning of a string/word (see Wikipedia "Substring"
+  ~ https://en.wikipedia.org/wiki/Substring#Prefix or
+  ~ https://math.stackexchange.com/questions/2190559/
+  is-there-standard-terminology-notation-for-the-prefix-of-a-word ).
+
+$)
+
+  $c prefix $.
+
+  $( Syntax for the prefix operator. $)
+  cpfx $a class prefix $.
+
+  ${
+    $d l s $.
+    $( Define an operation which extracts prefixes of words, i.e. subwords
+       starting at the beginning of a word.  Definition in section 9.1 of
```

Comment:
There is no definition number ?
Capitalize "Section"

Reply:
> There is no definition number ? 

Unfortunately, there is no definition number.

>Capitalize "Section"

OK

Reply:
Done